Detectors/Base/include/DetectorsBase/Track.h

```diff
@@ -197,10 +215,18 @@ namespace o2 {
       }
 
       //_______________________________________________________
-      inline Point3D<float> TrackParBase::GetXYZ() const {
-	std::array<float,3> xyz = { GetX(), GetY(), GetZ() };
-        Utils::RotateZ(xyz,GetAlpha());
-	return Point3D<float>(xyz[0],xyz[1],xyz[2]);
+      inline Point3D<float> TrackParBase::GetXYZGlo() const {
+	return Rotation2D(GetAlpha())( Point3D<float>(GetX(), GetY(), GetZ()) );
+      }
+
+      //_______________________________________________________
+      inline Point3D<float>  TrackParBase::GetXYZGloAt(float xk, float b, bool &ok) const {
+	//----------------------------------------------------------------
+	// estimate global X,Y,Z in global frame at given X
+	//----------------------------------------------------------------
+	float y=0.f,z=0.f;
+	ok = GetYZAt(xk,b,y,z);
+	return ok ? Rotation2D(GetAlpha())( Point3D<float>(xk,y,z) ) : Point3D<float>();
```

Comment:
formatting looks weird (here and in a couple of other places): for example the 2 space indentation rule.